src/Core/GrpcTrait.php

```diff
@@ -88,7 +88,7 @@ public function send(callable $request, array $args, $whitelisted = false)
     }
 
     /**
-     * Gets the default configuration for generated GAX clients.
+     * Gets the default configuration for generated ApiCore clients.
```

Comment:
This is on me for the odd description, but WDYT about `for GAPIC clients`?

Reply:
I think we actually want to start the process of moving away from the term `GAPIC` in our docs. What about `for generated clients` or `for generated API clients`?

Reply:
+1 to `for generated clients`.